README.md

```diff
@@ -28,7 +40,7 @@ Select 1:
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
-[] Selects all ul's of class dropdown, only if their children are exclusively li's
+[x] Selects all ul's of class dropdown, only if their children are exclusively li's
```

Comment:
It's sort-of the other way around. The answer is `Selects all li's which are directly inside a ul of class dropdown (children)`